### ISSUE TITLE
vmuser:wrong path for vmui

### ISSUE THREAD

**Issue by yimenghua** [VictoriaMetrics/operator#655]
controllers/factory/vmuser.go

vmauth fail to open vmui

wrong path for vmui

```go
func addVMSelectPaths(src []string) []string {
	return append(src, "/vmui",
		"/vmui/vmui",
		"/graph",
		"/prometheus/graph",
		"/prometheus/api/v1/label.*",
		"/graphite.*",
		"/prometheus/api/v1/query.*",
		"/prometheus/api/v1/rules",
		"/prometheus/api/v1/alerts",
		"/prometheus/api/v1/metadata",
		"/prometheus/api/v1/rules",
		"/prometheus/api/v1/series.*",
		"/prometheus/api/v1/status.*",
		"/prometheus/api/v1/export.*",
		"/prometheus/federate",
		"/prometheus/api/v1/admin/tsdb/delete_series",
	)
}
```

right path
```go
func addVMSelectPaths(src []string) []string {
	return append(src, "/vmui.*",
		"/vmui/vmui",
		"/graph",
		"/prometheus/graph",
		"/prometheus/api/v1/label.*",
		"/graphite.*",
		"/prometheus/api/v1/query.*",
		"/prometheus/api/v1/rules",
		"/prometheus/api/v1/alerts",
		"/prometheus/api/v1/metadata",
		"/prometheus/api/v1/rules",
		"/prometheus/api/v1/series.*",
		"/prometheus/api/v1/status.*",
		"/prometheus/api/v1/export.*",
		"/prometheus/federate",
		"/prometheus/api/v1/admin/tsdb/delete_series",
	)
}
```

**Comment by Haleygo** [comment]
What version did you use? Looks like it's already fixed after [v1.82.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases), details in https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1752

**Comment by yimenghua** [comment]
v1.89.1
I've read this issue https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1752 and tried many versions including v1.82.0.
I still can't open vmui through vmauth unless I change the path "/vmui" to "/vmui.*"

**Comment by Haleygo** [comment]
@yimenghua Yeah, you are right :)
Tested in vmauth, `/vmui/` couldn't match `/vmui` in [here](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/0bca7d49b9b7c104e0a6dc2f4106bf19e74e771e/app/vmauth/target_url.go#L32), caused 'missing route' error.
![image](https://github.com/VictoriaMetrics/operator/assets/39937150/10590e7d-b20f-4030-8941-2613310e085d)
<img width="708" alt="image" src="https://github.com/VictoriaMetrics/operator/assets/39937150/84716f25-6187-494c-b735-02d496e921a9">

**Comment by f41gh7** [comment]
Fixed at [v0.35.0](https://github.com/VictoriaMetrics/operator/releases/tag/v0.35.0) release
